### PR TITLE
Elasticsearch prod template

### LINKED ISSUE #46: Elasticsearch template
Add template which deploys jaeger with elasticsearch.

Reference to old discussion  https://github.com/jaegertracing/jaeger-openshift/pull/39#discussion_r141311275

### PR DESCRIPTION
**Pull request by pavolloffay**
Fixes #46

### CODE REVIEW

**Comment by pavolloffay** [comment]
@kevinearls could you please add jenkins job to test this template?

File: production/elasticsearch.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            env:
+              - name: NODE_NAME
+                value: master
+            volumeMounts:
+              - name: data
+                mountPath: /data
+            readinessProbe:
+              exec:
+                command:
+                - wget
```
Comment (pavolloffay) [inline production/elasticsearch.yml]: there is no curl in this image :/

**Comment by kevinearls** [comment]
@pavolloffay Sure, I'll start working on it today.

**Comment by pavolloffay** [comment]
Thanks, you probably needs this in advance, can you test it manually? We merge this and then CI setting?

**Comment by kevinearls** [comment]
@pavolloffay I've run the performance tests using this template and it works properly, so I think you can go ahead and merge it.  I am currently working on some updates to the tests and should be able to get those checked in within a day or so.

**Comment by pavolloffay** [comment]
@kevinearls thanks. @objectiser do you want to have a look? The template is basically the same as in k8s repo just with a different image

**Comment by objectiser** [comment]
@pavolloffay LGTM, but haven't tested. Ideally at some point would be good if both templates could use the same image, but fine for now.

**Comment by pavolloffay** [comment]
thanks, the problem is that official image is running it under the root. I will try to report it in elastic.

**Pull request merged**